src/content/faq/General.md

```diff
@@ -15,45 +15,39 @@ Soufflé candy lollipop [dragée cake chocolate](). Marzipan lemon drops sweet r
 
 ## Is GraphQL a database language like SQL?
 
-<!-- TODO -->
+No, but this is a common misconception. GraphQL is designed for APIs, not databases. It’s a specification for how to request data and it determines how that data is communicated between servers and clients.
```

Comment:
```suggestion
No, but this is a common misconception. GraphQL is designed for APIs, not databases. It’s a specification for how to request/modify data through API layer and it determines how that data is communicated between servers and clients.
```

Reply:
Decided to break it into two sentences, but kept your edits!